examples/PRReportGeneration.ps1

```diff
@@ -29,34 +31,11 @@ while($continueFetching)
         $prClosedData = [datetime]::ParseExact($prClosedDateInString.SubString(0,10),'yyyy-mm-dd', $null)
         $prId = $pr | Select-Object -ExpandProperty pullRequestId        
 
-        $bypassed = $false
-
-        $evaluationResults = az repos pr policy list --org $org --id $prId -o json| ConvertFrom-Json
-
-        foreach($evaluationResult in $evaluationResults)
-        {
-            $completedDate = $evaluationResult.completedDate
-            if($completedDate -eq $null)
-            {
-                $isBlockingText = $evaluationResult.configuration | Select-Object -ExpandProperty isBlocking
-                $isBlocking = [System.Convert]::ToBoolean($isBlockingText) 
-
-                if($isBlocking)
-                {
-                    $bypassed = $true
-                }
-            }
-        }
+        $prDetails = az repos pr show --org $org --id $prId -o json| ConvertFrom-Json
 
-        if($bypassed)
-        {
-            Write-Host "."
-            Write-Host "Bypassed PR is $($prId)"
-            $bypassedCount = $bypassedCount + 1
-        }
-        Else
+        if($prDetails.completionOptions.bypassPolicy)
         {
-            $nonByPassedCount = $nonByPassedCount + 1
+            $bypassPRsInfo = $bypassPRsInfo + "$($prId) is bypassed with Bypass reason $($prDetails.completionOptions.bypassReason) `n"
```

Comment:
we should also plan to add the name of the person who bypassed the PR. It is available in Unique name attribute in the response

Reply:
done